android/app/build.gradle

```diff
@@ -101,14 +101,25 @@ android {
     buildToolsVersion '25.0.1'
     defaultConfig {
         applicationId 'co.bitnation'
-        minSdkVersion 16
+        minSdkVersion 18
         targetSdkVersion 22
         versionCode 1
         versionName "1.0"
         ndk {
             abiFilters "armeabi-v7a", "x86"
         }
         vectorDrawables.useSupportLibrary = true
+        testBuildType System.getProperty('testBuildType', 'debug')  //this will later be used to control the test apk build type
+        missingDimensionStrategy "minReactNative", "minReactNative46" //read note
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    signingConfigs {
+        release {
+            storeFile file(System.getenv("HOME") + '/bitnation_android_keys_dev')
```

Comment:
@william2958 can you please make it just an Env var that point's to the key store file? The name of the var could be `BITNATION_ANDROID_KEY_STORE`